rtic-monotonics v2.0.2 not compiling

rtic-monotonics v2.0.2 fails to compile with this error: 
` error[E0599]: no method named `fetch_add` found for struct `portable_atomic::AtomicU32` in the current scope `
at rtic-monotonics-2.0.2\src\systick.rs:106:25 and rtic-monotonics-2.0.2\src\systick.rs:126:25

This project has the following config: 
` 
  [build]
  target = "thumbv6m-none-eabi"
  
  [unstable]
  build-std = ["core"]
`

--- a/rtic-monotonics/src/lib.rs
+++ b/rtic-monotonics/src/lib.rs
@@ -1,6 +1,11 @@
 //! In-tree implementations of the [`rtic_time::Monotonic`] (reexported) trait for
 //! timers & clocks found on commonly used microcontrollers.
 //!
+//! If you are using a microcontroller where CAS operations are not available natively, you might
+//! have to enable the `critical-section` or `unsafe-assume-single-core` feature of the
+//! [`portable-atomic`](https://docs.rs/portable-atomic/latest/portable_atomic/) dependency
+//! yourself for this dependency to compile.
+//!
 //! To enable the implementations, you must enable a feature for the specific MCU you're targeting.
 //!
 //! # Cortex-M Systick

